code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -164,28 +164,37 @@
 	icon_state_on = "freezer_1"
 	icon_state_open = "freezer-o"
 	max_temperature = T20C
-	min_temperature = 170
+	min_temperature = 170 //actual minimum temperature is defined by RefreshParts()
 	circuit = /obj/item/circuitboard/machine/thermomachine/freezer
+	
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer/on
+	on = TRUE
+	target_temperature = 73.15
```

Comment:
Change this to Init and do `target_temperature = min_temperature` instead, that way it'll be set to the correct temp if a mapper modifies the parts.

Reply:
I like that idea! Let me change the files.

Reply:
Okay done